### PR TITLE
Feature/461 add pagination getuserprojects endpoint

### PR DESCRIPTION
**Pull request by BartvanEijkelenburg**
## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I did not update API Controllers, if I did, I added/changed Postman tests
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [ ] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Run the project.

Go to swagger or Postman, log in as bob bob.

Select the Get User projects endpoint and try it out with the pagination options.

## Link to issue
Closes: #461

### CODE REVIEW

File: Postman/dex.postman_collection.json [hunk 1]
```diff
@@ -1640,6 +1640,20 @@
...
 						},
+						{
+							"name": "UserProjects",
+							"item": [
+								{
+									"name": "UserProject-GetProject-Self-Administrator",
+									"request": {
+										"method": "GET",
+										"header": [],
+										"url": null
```
Comment (MeesvanStraten) [inline Postman/dex.postman_collection.json:1651]: As we spoke about on Teams, all looks good except this url probably caused the tests to fail.

File: Postman/dex.postman_collection.json [hunk 2]
```diff
@@ -29201,6 +29215,20 @@
...
 						},
+						{
+							"name": "UserProjects",
+							"item": [
+								{
+									"name": "UserProject-GetProject-Self-Alumni",
+									"request": {
+										"method": "GET",
+										"header": [],
+										"url": null
```
Comment (MeesvanStraten) [inline Postman/dex.postman_collection.json:29226]: same as above ^^

**Review by MeesvanStraten — CHANGES_REQUESTED** [review]
see comments for postman tests.

**Review by MeesvanStraten — APPROVED** [review]

**Review by BasvanMil — APPROVED** [review]

**Review by DaveBouman — APPROVED** [review]
Look good

**Pull request merged**